RetailCoder.VBE/UI/RubberduckUI.resx

```diff
@@ -2052,4 +2052,13 @@ Would you like to import them to Rubberduck?</value>
     <value>Method '{0}' is an implementation of event '{1}.{2}'.  Rename event '{2}'?</value>
     <comment>0: Selected target Identifier; 1: Event Parent; 2: Event name</comment>
   </data>
+  <data name="GeneralSettings_EnableSourceControl" xml:space="preserve">
+    <value>Enable Source Control.  (Restart Rubberduck to take effect.)</value>
+  </data>
+  <data name="GeneralSettings_EnableSourceControlWarning" xml:space="preserve">
+    <value>Source Control is Experimental and is known to be troublesome.  Use at your own risk!</value>
+  </data>
+  <data name="GeneralSettings_ExperimentalFeatures" xml:space="preserve">
+    <value>Experimental Features:</value>
```

Comment:
If it's resurrected in its current form, I'd put it there, yes.

Don't make anything under "experimental features" SC-specific, as more things can (and likely *will*) end up in there.